src/client/app/Main.jsx

```diff
@@ -127,17 +124,25 @@ class Main extends Component {
           {
             (this.state.loggedIn)
               ? <span> Welcome, <strong>{this.state.user}</strong>!&nbsp;&nbsp;<a href="#" onClick={this.handleLogOut.bind(this)}>Log Out</a>&nbsp;&nbsp;</span>
-              :
-              <span><a href="#" onClick={this.handleLoggingIn.bind(this)}>Log In</a>&nbsp;&nbsp;
+              : <span><a href="#" onClick={this.handleLoggingIn.bind(this)}>Log In</a>&nbsp;&nbsp;
                 <a href="#" onClick={this.handleSigningUp.bind(this)}>Sign Up</a>&nbsp;&nbsp;</span>
           }
           <SearchBar handleSearch={this.handleSearch.bind(this)}/>
         </div>
         <div className="col-xs-12 container">
           <h3>Popular Items</h3>
           <PopularItems products={this.state.popular}/>
-          <FeaturedLists />
         </div>
+        {
+          (this.state.searchResults.length !== 0)
+            ? <div className="col-xs-12 container">
+              <SearchResults results={this.state.searchResults}/>
+            </div>
+            : <div className="col-xs-12 container">
+              <h3>Featured WishLists</h3>
+              <FeaturedLists />
```

Comment:
Do we want to only render <FeaturedLists /> when there are no search results or do we always want it on the page?  As written it wouldn't render after a successful search.

Reply:
Yeah I made it so that the featured lists are hidden once the search query returns results. Ideally we'd want to render it off to the side vertically, but currently it looks like the featuredList component has a fixed width so it wasn't possible to dynamically render it in its current iteration.